tests/test_huggingface.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+import numpy as np
+import tensorflow as tf
+from transformers import AutoTokenizer, TFAutoModel
+
+from finetune import SequenceLabeler
+from finetune.base_models.huggingface.models import HFBert, HFElectraGen, HFElectraDiscrim, HFXLMRoberta
+
+
+class TestHuggingFace(unittest.TestCase):
+    def setUp(self):
+        self.text = """
+        The quick brown fox jumps over the lazy dog. Il ne faut pas se fier aux apparences.
+        Año nuevo, vida nueva. Das ist mir Wurst.
```

Comment:
Might be worth including some multichar characters from the tokenizer test.https://github.com/IndicoDataSolutions/finetune/blob/c04a6d01ba8568c26c2d201c2dba5385027b0492/tests/test_utils.py#L35